src/timeline/TimelineModel.cpp

```diff
@@ -822,26 +869,26 @@ TimelineModel::markEventsAsRead(const std::vector<QString> &event_ids)
         }
 }
 
+template<typename T>
 void
-TimelineModel::sendEncryptedMessageEvent(const std::string &txn_id,
-                                         nlohmann::json content,
-                                         mtx::events::EventType eventType)
+TimelineModel::sendEncryptedMessage(mtx::events::RoomEvent<T> msg, mtx::events::EventType eventType)
```

Comment:
This may be a bad rebase?

Reply:
NO it isn't, I changed that api, rather than having two seperate fields for that I changed it to one single filed which takes message

Reply:
I see, makes sense